index.js

```diff
@@ -0,0 +1,11 @@
+import Clipboard from './src/clipboard';
+import ClipboardAction from './src/clipboard-action';
+import DelegateEvents from 'delegate-events';
+import TinyEmitter from 'tiny-emitter';
+
+// expose bundled libraries
+Clipboard.ClipboardAction = ClipboardAction;
+Clipboard.DelegateEvents = DelegateEvents;
+Clipboard.TinyEmitter = TinyEmitter;
```

Comment:
If you are bundling libraries, it is common practice to expose them so they can be accessed by library users.